tests/registry_disk_test.go

```diff
@@ -160,7 +160,7 @@ var _ = Describe("ContainerDisk", func() {
 				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskCirros))
 				for ind, volume := range vmi.Spec.Volumes {
 					if volume.ContainerDisk != nil {
-						vmi.Spec.Volumes[ind].ContainerDisk.Path = "/custom-disk/boot.img"
+						vmi.Spec.Volumes[ind].ContainerDisk.Path = "/custom-disk/cirros.img"
```

Comment:
why was this change made?

Reply:
Please see the dockerfile bellow. I was experimenting with negative test and it slipped into the PR. The cirros.img is the one used in dockerfile.

```dockerfile
FROM kubevirt/container-disk-v1alpha

LABEL maintainer="David Vossel <dvossel@redhat.com>"

# Add cirros
RUN curl https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img > /disk/cirros.img
RUN mkdir -p /custom-disk && cp /disk/cirros.img /custom-disk/cirros.img
```